content/recipes/swagger.md

```diff
@@ -563,6 +566,7 @@ Nest provides a set of short-hand **API response** decorators that inherit from
 - `@ApiNotAcceptableResponse()`
 - `@ApiRequestTimeoutResponse()`
 - `@ApiConflictResponse()`
+- `@ApiGatewayTimeoutResponse()`
```

Comment:
This one is duplicated now

Reply:
@kamilmysliwiec good catch. :see_no_evil: 
I have corrected it, it was another one that was missing. :bow: